### ISSUE TITLE
Loadouts appear to be broken on some bots

### ISSUE THREAD

**Issue by TheBlocks** [RLBot/RLBot#400]
Snek's loadout seems to be broken. It normally uses a flashy green boost but it's using a blue Datastream boost with the API version of RLBot.

**Comment by tarehart** [comment]
Good call! One factor is that we lost support for painted items when transitioning to the Psyonix API.

There may be another bug on top of that, though.

**Comment by NicEastvillage** [comment]
I have not tested thoroughly, but primary and secondary colors also have problems. For instance, blue team's purple is green ingame.
ref: https://github.com/RLBot/RLBot/wiki/Bot-Customization#colors

**Comment by tarehart** [comment]
Additionally we have confirmed reports of antennas being incorrect. Example: 

```
[Bot Loadout Orange]
team_color_id = 39
custom_color_id = 12
car_id = 523
decal_id = 583
wheels_id = 376
boost_id = 48
antenna_id = 0
hat_id = 0
paint_finish_id = 2066
custom_finish_id = 2066
engine_audio_id = 0
trails_id = 0
goal_explosion_id = 1971
```

Results in a saturn antenna, but should have given no antenna. On other loadouts we have reports of a dollar antenna appearing in lieu of no antenna.

**Comment by tarehart** [comment]
Regarding the paint colors, our mapping is based on this formula: https://jsfiddle.net/ufm5cyvk/

Which was an improvement over this original formula: https://jsfiddle.net/2x86zrj9/1

I don't know how blue team's purple could have been expressed as green on this basis, since we got pretty close to https://github.com/RLBot/RLBot/wiki/Bot-Customization#colors

**Comment by tarehart** [comment]
Blue team paint and the antenna issue are fixed in version 1.21.0!

I think everything else discussed here is explained by our lack of support for painted items, so I'm resolving.
